src/OpenApi/test/OpenApiGeneratorTests.cs

```diff
@@ -88,17 +88,17 @@ static void AssertCustomRequestFormat(OpenApiOperation operation)
         }
 
         AssertCustomRequestFormat(GetOpenApiOperation(
-            [Consumes("application/custom")] (InferredJsonClass fromBody) => { }));
+            [Consumes("application/custom")] ([FromQuery] int foo) => { }));
 
         AssertCustomRequestFormat(GetOpenApiOperation(
-            [Consumes("application/custom")] ([FromBody] int fromBody) => { }));
```

Comment:
Can we update this test just to assert there are no parameters, but keep the test route handlers the same?